test/SlowTests/Issues/RavenDB_187.cs

```diff
@@ -1,22 +1,21 @@
+using FastTests;
 using Raven.Abstractions.Data;
 using Raven.Json.Linq;
-using Raven.Tests.Common;
-using Raven.Tests.Common.Dto;
-
+using SlowTests.Core.Utils.Entities;
 using Xunit;
 
-namespace Raven.Tests.Issues
+namespace SlowTests.Issues
 {
-    public class RavenDB_187 : RavenTest
+    public class RavenDB_187 : RavenTestBase
     {
-        [Fact]
+        [Fact(Skip = "Delete marker is no longer supported")]
```

Comment:
Since we no longer support the notion of virtually deleted documents (see: http://issues.hibernatingrhinos.com/issue/RavenDB-187) we don't need to test for them.
This test should be removed.